modules/machine-vsphere-machines.adoc

```diff
@@ -42,7 +42,6 @@ Ensure that all virtual machine names across a vSphere installation are unique.
 .. Optional: On the *Select storage* tab, customize the storage options.
```

Comment:
Can we also remove this `Optional:` this has been a point of contention.

Reply:
Thanks, @Timdawson264 . I removed this sentence from both `machine-vsphere-machines.adoc` and `installation-vsphere-machines.adoc` files.